src/utils/files.ts

```diff
@@ -11,6 +11,34 @@ function validFileType(file: File, allowedTypes: readonly FileType[]) {
   return allowedTypes.some((type) => type.extension.includes(extension) && validUploadType(type.queryCall));
 }
 
+function isBoolean(strings: Set<string>) {
+  // This function tests whether both elements of an array belong to the string
+  // set.
+  const hasPair = ([a, b]: [string, string]) => strings.has(a) && strings.has(b);
+
+  // This is a list of two-element arrays containing valid false/true value
+  // pairs.
+  const candidates: Array<[string, string]> = [
+    ['0', '1'],
+    ['no', 'yes'],
+    ['No', 'Yes'],
+    ['NO', 'YES'],
+    ['n', 'y'],
+    ['N', 'Y'],
+    ['false', 'true'],
+    ['False', 'True'],
+    ['FALSE', 'TRUE'],
+    ['off', 'on'],
+    ['Off', 'On'],
+    ['OFF', 'ON'],
```

Comment:
Is On/Off ever actually used to represent boolean values? I've never seen it used this way.

Reply:
It's part of the [YAML spec](https://yaml.org/type/bool.html) :shrug: 

If we have the other YAML values here we should have On/Off too.